### DISCUSSION TITLE
How to decipher the files downloaded with "download-saves"?

### DISCUSSION THREAD

**Discussion by Chekote** [legendary-gl/legendary#575] [Q&A]
Howdy 👋 

I am trying to download my saves from Epic so I can transfer them to Steam versions of the game. I have successfully downloaded the files, but they are very cryptic. I was hoping they would match what the game saves, but it seems they are in some kind of proprietary Epic structure.

Is there any information on how to decipher these files and reconstruct the normal save files that my games expect?

Thanks in advance

**Comment by derrod** [comment]
What game are you talking about? Cloud saves on Epic work just like on Steam, where some or all files from a specific folder are uploaded. Legendary downloads them in the exact same structure they were uploaded in. If that doesn't match the Steam version perhaps the game does not use the same save game format for some reason.

**Reply by Chekote** [reply]
Far Cry 6 is the current one I am trying to restore. I think I got a little confused because it seems that maybe the files for each game are in their normal structure, but the games themselves have some kind of code name?

<img width="456" alt="image" src="https://github.com/derrod/legendary/assets/550486/3689d47f-ffdb-438a-8371-b75763633612">

So I'm not sure which one is Far Cry 6.

**Reply by derrod** [reply]
If you didn't specify a codename it'll download all available saves. You can see the app name for Far Cry in the `legendary list` command.
